project/BuildDependencies.scala

```diff
@@ -5,8 +5,9 @@ private object BuildDependencies {
 
   val compile: Seq[ModuleID] = dependencies(
     shared = Seq("com.iheart" %% "ficus" % "1.4.7"),
-    play25 = Seq("uk.gov.hmrc" %% "http-verbs" % "9.8.0-play-25" % Provided),
-    play26 = Seq("uk.gov.hmrc" %% "http-verbs" % "9.8.0-play-26" % Provided)
+    play25 = Seq("uk.gov.hmrc" %% "http-verbs-play-25" % "10.11.0" % Provided),
```

Comment:
Is there any impact on users of the library currently using play 2.5 and play 2.6?  We're OK with dependency updates, but just want to understand the impact/reason.